libraries/state_history/include/eosio/state_history/log.hpp

```diff
@@ -87,52 +87,39 @@ class state_history_log {
 
    template <typename F>
    void write_entry(const state_history_log_header& header, const chain::block_id_type& prev_id, F write_payload) {
-      try {
```

Comment:
This diff is from https://github.com/EOSIO/eos/pull/10332
Did you check this out to make sure that it doesn't need to be ported?

Reply:
I moved the catch to the plugin's `on_accepted_block` so non-appbase apps (e.g. cltester) can use the state_history lib